File: core/src/adictive/games/SquareWorld.java
package adictive.games;

import com.badlogic.gdx.graphics.OrthographicCamera;
import com.badlogic.gdx.utils.viewport.FillViewport;

public class SquareWorld {
    final int width = 32;
    final int height = 32;

    private OrthographicCamera camera;
    public final TiledMap tiledMap = new TiledMap();

    public SquareWorld() {
        camera = new OrthographicCamera();
        camera.setToOrtho(false);
        new FillViewport(width, height, camera);
        tiledMap.init(width, height);
    }

    public OrthographicCamera getCamera() {
        return camera;
    }

    public int getWidth() {
        return width;
    }

    public int getHeight() {
        return height;
    }
}

File: core/src/adictive/games/SquareWorld.kt
package adictive.games

import com.badlogic.gdx.graphics.OrthographicCamera
import com.badlogic.gdx.utils.viewport.FillViewport

class SquareWorld {
    val width = 32
    val height = 32

    val camera: OrthographicCamera
    val tiledMap = TiledMap()

    init {
        camera = OrthographicCamera()
        camera.setToOrtho(false)
        FillViewport(width.toFloat(), height.toFloat(), camera)
        tiledMap.init(width, height)
    }
}
